charts/cluster-security/templates/podsecuritypolicy.yaml

```diff
@@ -0,0 +1,82 @@
+# Based on the baseline PodSecurityPolicy from https://kubernetes.io/docs/concepts/policy/pod-security-policy/#recommended-practice.
+# This is the recommended approach to simplify upgrading to the PodSecurity admission controller by Kubernetes v1.25.
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: psp-baseline
+  annotations:
+    seccomp.security.alpha.kubernetes.io/allowedProfileNames: '*'
+spec:
+  privileged: false
+  # The moby default capability set, minus NET_RAW
+  allowedCapabilities:
+    - 'CHOWN'
+    - 'DAC_OVERRIDE'
```

Comment:
Is there a particular reason for granting [`DAC_OVERRIDE`](https://manpages.debian.org/bullseye/manpages/capabilities.7.en.html#CAP_DAC_OVERRIDE)? That should normally ring alarm bells.

Reply:
No particular reason other than taking the boilerplate PodSecurityPolicy (as described in the comment at the top of the file) as the basis which we can iterate on.